README.md

```diff
@@ -2,53 +2,57 @@
-    - **-d, --data** *Required if no bucket is passed* The data source to load. This can be a gdb, shp, GeoJSON, zip, or a directory of any of these which gets loaded recursively. Zips and GeoJSON can also be loaded from remote sources via URL.
-    - **-t, --transformer** *Default: transformers/[[data file basename]].js* The transformer to use. This converts state-specific data formats to our [Point Data Schema](https://github.com/cfpb/grasshopper/blob/master/docs/point_data_spec.md)
+  - **Local method**
+    - [Install node](https://nodejs.org/)
+    - [Install GDAL](http://trac.osgeo.org/gdal/wiki/DownloadingGdalBinaries)
+      - On OSX, instead of using a binary or building from source, you can [download homebrew](http://brew.sh/) and `brew install gdal`.
+    - [Install elasticsearch](https://www.elastic.co/downloads/elasticsearch)
+      - You can also point the loader to elasticsearch running on another machine.
+    - Run `npm install` from the project root
+    - Test with `npm test -- <CLI Options>`
+    - Run with `./index.js <CLI Options>`
+
+  - **Docker method**
+    - [Install Docker](https://docs.docker.com/installation/#installation)
+    - Build an image:
+      `docker build --rm --tag=<image-name>:<tag-name> .`
+    - Test the image:
+      `./docker-test  <image-name>:<tag-name> <CLI Options>`
+    - Run the image:
+      `./docker-run <image-name>:<tag-name> <CLI Options>`
+      - These scripts assume there is an aws credentials file at `~/.aws/credentials` if using S3 to backup data.
+      - When using boot2docker, elasticsearch running on the host machine (eg, your Mac) can be accessed at 10.0.2.2 and elasticsearch running in a container with port 9200 shared can be accessed at 192.168.59.103.
+
+## CLI Options
+  - The loader is a command-line application, run by invoking `./index.js` with the following options:
+    - **-f, --file** *Required* A json data file that contains the collected data endpoints and field mappings of state data. `./data.json` should be used to load all known state data.
+    - **-m, --match** A string or regular expression that the names from the <file> must contain or match. Can be used to load just a few items from a large metadata file.
     - **-h, --host** *Default: localhost* The elasticsearch host. If no argument is provided and a linked elasticsearch Docker container exists, will use its IP.
     - **-p, --port** *Default: 9200* The elasticsearch port. If no argument is provided and a linked elasticsearch Docker container exists, will use its lowest exposed port.
+    - **-a, --alias** *Default: address* The elasticsearch alias to an internally created index. This what queries should be run against once data is loaded.
+    - **-t, --type** *Default: point* The elasticsearch type (or mapping) within the alias
     - **-l, --log** *Default: debug* The elasticsearch log level
```

Comment:
It looks like [Winston's log levels](https://github.com/winstonjs/winston#using-logging-levels) match up with the standard `debug`, `info`, `warn`, `error`, etc. as most logging libs.  Perhaps you can just have a single `--log` arg for both, and allow a `off` value to act as the current `-q`.  This of course goes against the usual cli `-q` convention, _and_ doesn't get you separate log levels for app and ES...but it would be consistent.

Looks like you can even setup winston to use [custom log levels](https://github.com/winstonjs/winston#using-custom-logging-levels), so you _make_ them match up if they don't quite now.